tests/lct_test.py

```diff
@@ -20,12 +20,94 @@
 ]
 
 
-def test_lct_signal():
+class _MultiCases:
```

Comment:
Lambdas are harder to read. It's nice to name the difference cases as functions. I did a few things including sorting the methods. This binds the name to the case to the file. I don't think the directories needs to be named "case_x". Rather, just "x.in" and "x.out"